discover/portal/articles/04-forms/00-intro.markdown

```diff
@@ -18,23 +18,28 @@ two applications for building forms to collect precisely the data you need:
 
 **Kaleo Forms:** If you're a Liferay Digital Enterprise customer, there's a
 third form building tool in @product@ called [Kaleo
-Forms](/discover/portal/-/knowledge_base/7-0/workflow-forms). It integrates form
+Forms](https://customer.liferay.com/documentation/7.0/admin/-/official_documentation/portal/workflow-forms). It integrates form
 building with workflow to create form based business processes, like a
 Conference Room Checkout Form, or a Support Ticket Process so support tickets go
 through the proper channels on their way to resolution. Read more about Kaleo
 Forms in the [workflow
-section](/discover/portal/-/knowledge_base/7-0/using-workflow).
+section](https://customer.liferay.com/documentation/7.0/admin/-/official_documentation/portal/workflow-forms).
```

Comment:
Have you changed your mind and have decided to refer the **workflow-forms** section instead of the **using-workflow** section?

Reply:
@nato yes, I think it's better to use an absolute link to the customer portal here. Community users will understand they can't see the material because they are not customers, instead of being confused by an Article Not Found message, since the Kaleo Forms article isn't on LDN.

Reply:
I see your concern and it's valuable! The goal of my previous comment was just ask if you've intentionally changed the article from [https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/using-workflow](https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/using-workflow) to [https://customer.liferay.com/documentation/7.0/admin/-/official_documentation/portal/workflow-forms](https://customer.liferay.com/documentation/7.0/admin/-/official_documentation/portal/workflow-forms), i.e. not just change the place where the content is published, but also the referenced article.

Reply:
Oh I see what you mean, @natocesarrego . Yes I did intend to link directly to Kaleo Forms instead of the intro to the workflow section.